packages/webapp/src/pages/Environment/Settings.tsx

```diff
@@ -988,6 +1058,111 @@ export const EnvironmentSettings: React.FC = () => {
+                                                    autoComplete="new-password"
+                                                    type="password"
+                                                    onMouseEnter={(e) => (e.currentTarget.type = 'text')}
+                                                    onMouseLeave={(e) => {
+                                                        if (document.activeElement !== e.currentTarget) {
+                                                            e.currentTarget.type = 'password';
+                                                        }
+                                                    }}
+                                                    onFocus={(e) => (e.currentTarget.type = 'text')}
+                                                    onBlur={(e) => (e.currentTarget.type = 'password')}
+                                                    className="border-border-gray bg-bg-black text-text-light-gray focus:ring-blue block h-11 w-full appearance-none rounded-md border text-base placeholder-gray-600 shadow-sm focus:border-blue-500 focus:outline-none"
+                                                />
+                                                <button
+                                                    onClick={() => handleRemoveOtlpHeader(key)}
+                                                    className="flex hover:bg-hover-gray text-white h-11 ml-4 px-4 pt-3 text-sm"
+                                                    type="button"
+                                                >
+                                                    <TrashIcon className="flex h-5 w-5 text-white" />
+                                                </button>
+                                            </div>
+                                        ))}
+                                        <div className="flex justify-end mt-4">
+                                            <button
+                                                onClick={handleAddOtlpHeader}
+                                                className="hover:bg-hover-gray bg-gray-800 text-white flex h-11 rounded-md px-4 pt-3 text-sm mr-4"
+                                                type="button"
+                                            >
+                                                Add OpenTelemetry Endpoint Header
+                                            </button>
+                                            <button type="submit" className="hover:bg-gray-200 bg-white text-gray-700 flex h-11 rounded-md px-4 pt-3 text-sm">
+                                                Save OpenTelemetry Endpoint Headers
+                                            </button>
+                                        </div>
+                                    </form>
+                                </div>
+                            )}
+                        </div>
```

Comment:
Would be nice to have an example in the tooltip or better in the placeholder of the input

Reply:
good point. will do